app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/artist/ArtistScreen.kt

```diff
@@ -74,6 +77,7 @@ private fun ArtistsScreen(
                 }
             ) {
                 Column(horizontalAlignment = Alignment.CenterHorizontally) {
+//                    Spacer(modifier = Modifier.width(100.dp)) // added Spacer
```

Comment:
What's the logic behind commenting this and adding it for review?

Reply:
oh that i forgot to remove it. i am trying to add space as in artist the left box is nearly touching the edge. i will remove it